test/functorch/test_aotdispatch.py

```diff
@@ -1194,66 +1193,6 @@ def forward(self, primals_1):
     #     return [sin, copy]""",
     #         )
 
-    def test_input_mutation_storage_resize_down_and_set_(self):
-        # Meant to mimic ppFSDP
```

Comment:
https://github.com/pytorch/pytorch/blob/4e89e3954d823f4b250e189fb3886eb3cb49af75/test/inductor/test_distributed_patterns.py#L17-L107 is a better mimicking of ppFSDP, I think we can just remove this test now.